### ISSUE TITLE
Support of PHP versions?

### ISSUE THREAD

**Issue by stephanvierkant** [KnpLabs/KnpMenuBundle#449]
I see this library still supports PHP 7.2, which is End of Life (EOL) for a while. Even 7.3 is EOL as of today.

What's the policy about PHP support? The only [supported versions](https://www.php.net/supported-versions.php) are 7.4, 8.0 and 8.1

In my opinion libraries should drop support of PHP versions that are end of life, because it makes contributing to this library much easier.

**Comment by garak** [comment]
I agree, but I think it doesn't make much sense to drop unsupported version in the bundle if they're still supported in the main library.

Do you mind opening a PR to drop PHP 7.3 from [KnpMenu](https://github.com/KnpLabs/KnpMenu)? 7.2 was already dropped.

**Comment by stephanvierkant** [comment]
See https://github.com/KnpLabs/KnpMenu/issues/357

**Comment by Chris53897** [comment]
This Issue could be closed as done.
Or meanwhile it could be raised to PHP 8.0 ?

**Comment by Chris53897** [comment]
This could be closed as completed.
